build-mbl/support/file_util.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
```

Comment:
that's taken from thud-gateway-bsp branch?

Reply:
Correct.

Reply:
In the thud-gateway-bsp branch it is at the top level. I've added some folder structure to not add complications to the top level.